test/js/workerd/html-rewriter.test.js

```diff
@@ -548,7 +547,7 @@ const request_types = ["/", "/gzip", "/chunked/gzip", "/chunked", "/file", "/fil
     it(`works with ${protocol} fetch using ${url}`, async () => {
       const server = protocol === "http" ? http_server : https_server;
       const server_url = server.url;
-      const res = await fetch(`${server_url}${url}`, { tls: { rejectUnauthorized: false } });
+      const res = await fetch(`${server_url}${url.slice(1)}`, { tls: { rejectUnauthorized: false } });
```

Comment:
Is the URL normalization different on Windows versus Posix? I'm confused.

Reply:
`server.url` makes a string like `http://localhost:8000/`
`url` is a string like `/chunked`
so the url that showed up in the logs was `http://localhost:8000//chunked` with a double slash

Reply:
should we just use `server.url.origin` instead of `server.url`? (server.url is a `URL` object)

Reply:
that would also work great, I can go with that.